.travis.yml

```diff
@@ -69,63 +69,68 @@ jobs:
     - script: make edgesite_cross_build_v7
       name: "cross build edgeite for armv7"
 
-    - stage: "unit tests"
+    - stage: "Test Build on arm64"
+      script: make
+      arch: arm64
+      name: "builds cloud and edge components"
+    - script: make bluetoothdevice
+      arch: arm64
+      name: "build bluetooth_device"
+    - script: make edge_small_build
+      arch: arm64
+      name: "build small edge version"
+
+    - stage: "unit tests on amd64"
       script: make edge_test
       arch: amd64
-      name: "unit test edge on amd64"
+      name: "unit test edge"
     - script: make edge_integration_test
       arch: amd64
-      name: "integration test edgeo on amd64"
+      name: "integration test edge"
     - script: make cloud_test
       arch: amd64
-      name: "unit test cloud on amd64"
+      name: "unit test cloud"
+
+    - stage: "unit tests on arm64"
+      script: make cloud_test
+      arch: arm64
+      name: "unit test cloud"
```

Comment:
Could we add `edge_test` here?

Reply:
so far, we cannot,  pls see https://travis-ci.com/chendave/kubeedge/jobs/255951848

Reply:
ok, it seems there's some errs about database operation?

Reply:
I will dig into it, might also related with so called "privileged" requirement around lxd.

Reply:
OK, thanks.